dubbo-dependencies-bom/pom.xml

```diff
@@ -96,7 +96,7 @@
         <javassist_version>3.29.2-GA</javassist_version>
         <bytebuddy.version>1.14.5</bytebuddy.version>
         <netty_version>3.2.10.Final</netty_version>
-        <netty4_version>4.1.92.Final</netty4_version>
+        <netty4_version>4.1.94.Final</netty4_version>
```

Comment:
Why change the version?

Reply:
> Why change the version?

4.1.92.Final has vulnerability ,  CVE-2023-34462. Severity: MEDIUM
refer : https://mvnrepository.com/artifact/io.netty/netty-handler

I will rollback it , hope dubbo community will upgrade it later .